lib/scarpe/display_service.rb

```diff
@@ -80,7 +80,10 @@ def subscribe_to_event(event_type, event_name, event_target, &handler)
 
       # TODO: add more display service types, use an env var to switch
       def display_services
-        @service_list ||= [WebviewDisplayService.new]
+        @service_list ||= [
+          WebviewDisplayService.new,
+          GlimmerLibUIDisplayService.new,
```

Comment:
If you do this, it's going to create *both* Webview and Glimmer. As long as that's intended, cool. You can see my other PR for how to do this with an env var.